README.md

```diff
@@ -3,7 +3,6 @@
 [![GitHub release](https://img.shields.io/github/release/tomv564/LSP.svg)](https://github.com/tomv564/LSP/releases)
 [![Build Status](https://travis-ci.org/tomv564/LSP.svg?branch=master)](https://travis-ci.org/tomv564/LSP)
 [![Coverage Status](https://coveralls.io/repos/github/tomv564/LSP/badge.svg?branch=master)](https://coveralls.io/github/tomv564/LSP?branch=master)
-[![Join the chat at https://gitter.im/SublimeLSP/Lobby](https://badges.gitter.im/SublimeLSP/Lobby.svg)](https://gitter.im/SublimeLSP/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Is it not possible to have a fancy badge to a discord channel?

Reply:
seems like it should be possible: https://gist.github.com/tterb/982ae14a9307b80117dbf49f624ce0e8#discord

Reply:
I have looked but haven't found one.

But IMO personally those are more form than function because I never notice them (or learned to ignored). I would rather search for relevant words when looking for a relevant chat, than looking for it in those badges.

Reply:
I use that badge all the time to get to the chat...

Reply:
like this? [![Discord](https://img.shields.io/discord/280102180189634562)](https://discordapp.com/channels/280102180189634562/645268178397560865)